test/unit/UFragmentsPolicy.js

```diff
@@ -208,45 +205,45 @@ contract('UFragments:setDeviationThreshold:accessControl', function (accounts) {
   });
 });
 
-contract('UFragmentsPolicy:setRebaseLag', async function (accounts) {
-  let prevLag;
-  before('setup UFragmentsPolicy contract', async function () {
-    await setupContracts();
-    prevLag = await uFragmentsPolicy.rebaseLag.call();
-  });
-
-  describe('when rebaseLag is more than 0', async function () {
-    it('should setRebaseLag', async function () {
-      const lag = prevLag.plus(1);
-      await uFragmentsPolicy.setRebaseLag(lag);
-      (await uFragmentsPolicy.rebaseLag.call()).should.be.bignumber.eq(lag);
-    });
-  });
-
-  describe('when rebaseLag is 0', async function () {
-    it('should fail', async function () {
-      expect(
-        await chain.isEthException(uFragmentsPolicy.setRebaseLag(0))
-      ).to.be.true;
-    });
-  });
-});
-
-contract('UFragments:setRebaseLag:accessControl', function (accounts) {
-  before('setup UFragmentsPolicy contract', setupContracts);
-
-  it('should be callable by owner', async function () {
-    expect(
-      await chain.isEthException(uFragmentsPolicy.setRebaseLag(1, { from: deployer }))
-    ).to.be.false;
-  });
-
-  it('should NOT be callable by non-owner', async function () {
-    expect(
-      await chain.isEthException(uFragmentsPolicy.setRebaseLag(1, { from: user }))
-    ).to.be.true;
-  });
-});
+// contract('UFragmentsPolicy:setRebaseLag', async function (accounts) {
```

Comment:
You can just replicate the same tests for `setRebaseFunctionGrowth` and `setRebaseFunctionUpperPercentage`

Reply:
That's the plan